Library/Formula/mono.rb

```diff
@@ -41,10 +51,11 @@ def install
   end
 
   test do
+    ohai "Remember to set MONO_GAC_PREFIX=\"#{HOMEBREW_PREFIX}\" to finalize the installation."
```

Comment:
This should be in ` def caveats ` I presume, if you want/need the end user to see it?